django/applications/catmaid/control/importer.py

```diff
@@ -275,7 +275,7 @@ def find_project_folders(image_base:str, path:str, filter_term) -> Tuple[List, D
                         url_dir = url_dir.replace("\\", "/")
                     project_url = urljoin(image_base, url_dir)
                     # Expect a YAML file
-                    yaml_data = yaml.load_all(open(info_file), Loader=yaml.FullLoader)
+                    yaml_data = yaml.load_all(open(info_file), Loader=yaml.SafeLoader)
```

Comment:
Apparently recent versions of the yaml loader may be missing this loader?

Reply:
That's surprising. Their changelog mentions it as added in version 5.1: https://pyyaml.org/wiki/PyYAML#history.  I can also import it locally. Maybe your `PyYAML` package isn't updated?

Reply:
You're right. In my case I actually didn't have it installed at all, just had (very dated) type headers installed separately.

Reply:
reverted

Reply:
The warning I was following was from an omission fixed upstream 2 weeks ago (no release since):
https://github.com/python/typeshed/commit/5c94b5875209329791791e54c0599273ac6ba2c0